app/database_etl/tableau_data_connector/table_generator.py

```diff
@@ -10,10 +10,13 @@
 load_dotenv()
 
 
-def upload_analyze_csv():
+def upload_analyze_csv(canadian_data):
```

Comment:
could probably find a cleaner/more generic way to do this but given that you're rushing rn I'll let you leave this as a todo